microsetta_interface/implementation.py

```diff
@@ -784,22 +783,39 @@ def post_account_details(*, account_id=None, body=None):
     return _refresh_state_and_route_to_sink(account_id)
 
 
+@prerequisite([ACCT_PREREQS_MET])
+def check_duplicate_source_name_email(*, account_id=None, body=None):
+    has_error, email_check_output, _ = ApiRequest.post(
+        "/accounts/{0}/check_duplicate_source".format(account_id), json=body)
+    if has_error:
+        return consent_output
```

Comment:
consent_output isn't defined anywhere in this function - did you mean email_check_output?